### ISSUE TITLE
cannot set aliases when creating an item

### ISSUE THREAD

**Issue by giorgiobevilacqua** [maxlath/wikidata-edit#22]
Hi!
I'd need to set aliases when creating a new item, like this:
```
      wdEdit.entity.create({
          labels: { en: obj.LABEL_MIMO},
          aliases: { en: obj.LABEL_GUIZZI, it: obj.TAXON_GUIZZI_ITA_ORIG },
          descriptions: { en: 'class of musical instruments', it: 'classe di strumenti musicali' },
          claims: myClaims
     };
```

but I get the following error:
```
TypeError: value.map is not a function
    at Object.keys.forEach.lang (/home/zummo/Scrivania/synapta/mps-bot/node_modules/wikidata-edit/lib/entity/format.js:13:27)
    at Array.forEach (native)
    at Object.values (/home/zummo/Scrivania/synapta/mps-bot/node_modules/wikidata-edit/lib/entity/format.js:8:25)
    at module.exports.data (/home/zummo/Scrivania/synapta/mps-bot/node_modules/wikidata-edit/lib/entity/validate_and_format_data.js:25:34)
    at Promise (/home/zummo/Scrivania/synapta/mps-bot/node_modules/wikidata-edit/lib/entity/edit.js:17:22)
    at Promise (<anonymous>)
    at _validateAndFormatData (/home/zummo/Scrivania/synapta/mps-bot/node_modules/wikidata-edit/lib/entity/edit.js:15:10)
    at Object.data [as edit] (/home/zummo/Scrivania/synapta/mps-bot/node_modules/wikidata-edit/lib/entity/edit.js:7:12)
    at createItem (/home/zummo/Scrivania/synapta/mps-bot/bot.js:128:21)
    at askWikidata (/home/zummo/Scrivania/synapta/mps-bot/bot.js:38:5)
    at callWikidata (/home/zummo/Scrivania/synapta/mps-bot/bot.js:27:5)
    at Object.<anonymous> (/home/zummo/Scrivania/synapta/mps-bot/bot.js:205:1)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
(node:21346) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): TypeError: Cannot read property 'error' of undefined
(node:21346) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by maxlath** [comment]
the function was expecting an array of aliases, 0a6a760 fixes it to also accept single strings.
patch published in `v2.3.1` :)

**Comment by maxlath** [comment]
oh, sry I thought it did it ^^, done!
